src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/CodeGeneration/MethodGenerator.cs

```diff
@@ -228,12 +227,24 @@ private static SyntaxList<TypeParameterConstraintClauseSyntax> GenerateDefaultCo
                 if (!referencedTypeParameters.Contains(typeParameter))
                     continue;
 
-                var constraint = typeParameter switch
+                TypeParameterConstraintSyntax? constraint;
```

Comment:
1. odd that this is nullable.
2. i do prefer the old switch form.  but i'm ok with if/else as well.

Reply:
> odd that this is nullable.

Probably used the "Use explicit type" refactoring. At least that's how I would end up with it being declared nullable.